lesson_03/homework_03.py

```diff
@@ -57,18 +97,38 @@
 Торт            1           350 грн
 Вода            3           21 грн
 """
+pizza_xl = 274
+pizza_m = 218
+juice = 35
+cake = 350
+water = 21
+total_bill = 4*pizza_xl + 2*pizza_m + 4*juice + cake + 3*water
+print('Щоб дізнатися скільки Іринка витратить на всі продукти, треба перемножити ціна них на їх вартість:\n'
+      f'Піца велика, 4шт = {pizza_xl}грн\n'
+      f'Піца середня, 2шт = {pizza_m}грн\n'
+      f'Сік, 4шт = {juice}грн\n'
+      f'Торт, 1шт = {cake}грн\n'
+      f'Вода, 3шт = {water}грн')
+print(f'Тоді повна сума до сплати буде - {total_bill}грн')
+print()
 
 
-# task 09
+print('task 09')
 """
 Ігор займається фотографією. Він вирішив зібрати всі свої 232
 фотографії та вклеїти в альбом. На одній сторінці може бути
 розміщено щонайбільше 8 фото. Скільки сторінок знадобиться
 Ігорю, щоб вклеїти всі фото?
 """
+photos_quantity = 232
+placed_quantity_on_page = 8
+total_pages = photos_quantity / placed_quantity_on_page
+total_sheet = total_pages / 2
```

Comment:
```suggestion
total_sheet = total_pages // 2
```
і не треба math.ceil

Reply:
@o-panchenko 
Не згоден, бо розраховуючи потребуючі аркущі, нам важливо отримати цифру кратну 2. Бо аркуш не може містити менше ніж 2 сторінки. 
В цій ситуації розрахунок виводить потребу у 14,5 аркушів, чого неможливо здійснити.
Тому я і використав тут math.ceil так як // повертає 14(тоді нам не вистачить на 232 фото), що не вірно бо нам потрібно 15 листів(29 аркушів)

Reply:
total_sheet = total_pages // 2  +  total_pages % 2
правда так теж виходить